app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
```

Comment:
- [ ] [REQUIRED] Great effort on the model associations! However, I noticed that you added explicit foreign keys (foreign_key: 'user_id' and foreign_key: 'post_id') to the belongs_to associations. In the original specification and ERD diagram, these associations were intended to be handled without explicit foreign keys.

For better adherence to the design and consistency with the project requirements, I recommend removing the explicit foreign keys.